float64 steering_angle  # Position to move to in radians ( -pi/2 -> pi/2)
float32 throttle_cmd    # Speed to move at (0 -> 1)
int8    throttle_mode   # 1: Stop, 2: Forward, 3: Backwards
